src/modules/navigator/precland.cpp

```diff
@@ -328,6 +321,8 @@ PrecLand::run_state_final_approach()
 void
 PrecLand::run_state_search()
 {
+	PX4_INFO("Precland: searching for target");
```

Comment:
Search is run on every landing_target_pose as long as
` _target_pose_updated && _target_pose_valid && _target_pose.abs_pos_valid` returns false.

Wich is dependent on the Kalman filter throwing an error. This might happen about every 10 seconds.

Reply:
I removed it. This Info does not give any new information to the user.